Cargo.toml

```
@@ -17,6 +17,9 @@ exclude = ["/.*"]
 [package.metadata.docs.rs]
 rustdoc-args = ["--cfg", "docsrs"]
 
+[lints.rust]
+unexpected_cfgs = { level = "warn", check-cfg = ['cfg(docsrs)', 'cfg(polling_test_poll_backend)', 'cfg(polling_test_epoll_pipe)'] }
```

```suggestion
unexpected_cfgs = { level = "warn", check-cfg = ['cfg(polling_test_poll_backend)', 'cfg(polling_test_epoll_pipe)'] }
```

cfg(docsrs) is an official cfg set by docs.rs. https://github.com/rust-lang/cargo/blob/431db31d0dbeda320caf8ef8535ea48eb3093407/src/cargo/core/compiler/mod.rs#L1351